.eslintrc

```diff
@@ -1,10 +1,43 @@
 extends: 'airbnb-base'
 env:
-  node: false
-  browser: true
-  commonjs: true
+  node: true
+  browser: false
 rules:
 
+  # Forces alignment of objects values and colons of both single line objects
+  # and multiLine objects. This promotes smaller diffs.
+  key-spacing:
+    - 2
+    - singleLine:
+        beforeColon: false
+        afterColon: true
+      multiLine:
+        beforeColon: true
+        afterColon: true
+        align: colon
```

Comment:
I don't think that this promotes smaller diffs. Because each line depends on the others.
E.g: If you have an object with 10 aligned keys and you add a new key that is larger than all the others, you have a 10 lines diff instead of one!

IMO we should not align keys on multiline objects.